calc.c

```diff
@@ -102,10 +102,12 @@ OpDef infix[] = {
 	{NULL, NULL},
 };
 
+Num op_readchar(Str* str) { return (Num)(char)*((*str)++); }
```

Comment:
make sure you check if you're at the end of the string

Reply:
like this?
https://github.com/12Me21/wscalc/pull/1/commits/4bde67e13afd91f8ad2a8023e1c437209523f0ef

Reply:
yeah